apps/tangle-dapp/data/restaking/useRestakingProfile.ts

```diff
@@ -31,19 +28,7 @@ const useRestakingProfile = (address?: string) => {
   }, [ledgerOpt]);
 
   const totalRestaked = useMemo(
```

Comment:
@AtelyPham I removed this code because I think the unit should be unified. Feel free to give me reasons why you were adding this check when implementing the restaking page